README.md

```diff
@@ -69,7 +55,7 @@ words.each do |word|
 end
 ```
 
-Snippet 6 - Big O:
+Snippet 6 - Big O: O(1) - using puts & array length is defined
```

Comment:
Hmm... on further thought, I see why you said O(1) with the statement "array length is defined" but we shouldn't assume that the array length is immutable, which is why it's not O(1)